apps/backoffice-v2/src/pages/Entity/components/Case/Case.Documents.module.css

```diff
@@ -0,0 +1,3 @@
+:global(.ReactCrop__child-wrapper) {
+  width: 100% !important;
+}
```

Comment:
Putting this on the ReactCrop component className prop should work and we won't need a new file, the global scope, or breaking out of Tailwind. '[&>.ReactCrop__child-wrapper]:!w-full'

Reply:
Good point, thanks